bin/alluxio

```diff
@@ -11,7 +11,7 @@
 #
 
 function printUsage {
-  echo "Usage: alluxio COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo "Usage: alluxio [-debug] COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
```

Comment:
I think it's better not to include this flag here since most users won't know about `ALLUXIO_USER_DEBUG_JAVA_OPTS`, and will expect to get more information when they use `-debug`.